hasher-matcher-actioner/hma-lite/hmalite/app.py

```diff
@@ -5,20 +5,28 @@
 from threatexchange.signal_type import index, pdq_index
 from threatexchange.hashing.pdq_hasher import pdq_from_file
 import os
+import os.path
+import sys
 import csv
 
+CONFIG_ENV = 'HMALITE_CONFIG_FILE'
+
 app = Flask(__name__)
 app.register_blueprint(matcher_api, url_prefix="/v1/hashes")
 
-# TODO Move to APP Config
-UPLOADS_FOLDER = "hmalite/uploads"
-INDEX_FOLDER = "hmalite/data"
-INDEX_FILENAME = "data.index"
+
+if app.config["ENV"] == "production":
+    app.config.from_object("hmalite.config.HmaLiteProdConfig")
+else:
+    app.config.from_object("hmalite.config.HmaLiteDevConfig")
+app.config.from_envvar('HMALITE_CONFIG_FILE', silent=True)
 
 
 @app.route("/")
 def index():
-    files = os.listdir("hmalite/data/")
+    files = []
+    if os.path.exists(app.config["INDEX_FILE"]):
+        files = [app.config["INDEX_FILE"]]
```

Comment:
@BarrettOlson - Did this because you said you were planning on optimizing to only use a single index file